### PR TITLE
Fix mistake in menu group specification.

### PR DESCRIPTION
**Pull request by prabhuramachandran**
Fixes https://github.com/enthought/mayavi/issues/1182

### CODE REVIEW

**Review by rahulporuri — APPROVED** [review]
This looks similar to other fixes we made recently.

**Comment by prabhuramachandran** [comment]
@rahulporuri -- thanks, not sure what the pip errors are here but requesting someone to please merge this when possible.

**Comment by mdickinson** [comment]
We do need to get to the bottom of the CI issues before we merge, even if that's just verifying that the same issue exists on the main branch, so that it's independent of this PR.

Unfortunately I won't have bandwidth for that in the near future.

File: envisage/ui/workbench/default_action_set.py
Comment (mdickinson) [inline envisage/ui/workbench/default_action_set.py:29]: Style: Let's please use black-style 4-space indents here and below.
```python
groups = [
    Group(id="OpenGroup"),
    ...
]
```

File: envisage/ui/workbench/default_action_set.py
Comment (mdickinson) [inline envisage/ui/workbench/default_action_set.py:14]:
```suggestion
from envisage.ui.action.api import Action, ActionSet, Group, Menu
```

Comment (prabhuramachandran) [reply envisage/ui/workbench/default_action_set.py]: Thanks, fixed the stylistic issues.

**Comment by mdickinson** [comment]
I ran the test-with-pypi workflow on main; indeed it's failing in the same way: https://github.com/enthought/envisage/actions/runs/3264826259

**Comment by mdickinson** [comment]
The issue appears to be coming from Pyface: https://github.com/enthought/pyface/issues/1163

**Comment by mdickinson** [comment]
Changes look good. But this is a bugfix, so it should really have a regression test. @prabhuramachandran: is there something that could be stolen from Mayavi that would act as a suitable regression test?

**Comment by prabhuramachandran** [comment]
I have added the smallest possible test, a simple import catches the error. I hope this is good enough?

**Comment by prabhuramachandran** [comment]
@mdickinson -- I added probably the most trivial of tests. It fails without the changes and passes with them.

**Comment by prabhuramachandran** [comment]
Looks like I may have messed up something in the commit, I am not sure but the test errors seem unrelated here.

**Comment by prabhuramachandran** [comment]
@mdickinson -- is there anything else needed for this?

File: envisage/tests/test_workbench.py
Comment (mdickinson) [inline envisage/tests/test_workbench.py:16]:
```suggestion
        import envisage.ui.workbench.default_action_set  # noqa: F401
```

**Comment by mdickinson** [comment]
> is there anything else needed for this?

Just the flake8 fix, I think (see https://github.com/enthought/envisage/pull/485/commits/754bd86ea9cfbb5a896c633394dfe9e145d4e11d). I'll merge when/if the  checks pass.

**Comment by mdickinson** [comment]
Merged. Thanks for the fix.

**Pull request merged**
